db/migrate/20190707093458_create_plants.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePlants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :plants do |t|
+      t.string :name
+      t.integer :irrigation_frequency
```

Comment:
This will be measured in days, right? :smile:

Reply:
Yes 👍🏻

Reply:
Maybe it's a good idea to rename this column to `:irrigation_frequency_in_days`? What do you think @sleaz0id? :relaxed:
```suggestion
      t.integer :irrigation_frequency_in_days
```

Reply:
Absolutely, great idea 💪.